azure-simple/packer/ansible-is/scripts/update_README.md

```diff
@@ -0,0 +1,19 @@
+# Continuous Update Delivery for WSO2 Identity Server
+
+### Prequisites
+* Product packs should be provided in the `/files/packs` directory
+
+### Usage
+While executing the update script, provide the profile name. The pack corresponding to the profile will begin updating.
+```bash
+./update.sh -p <profile-name>
```

Comment:
We can say the relative path from the root directory.

Reply:
I think this script needs to run from inside the 'scripts' directory because the 'update.sh' script referring the current directory to find the pack directory ': ${packs_dir:=$(pwd)/../files/packs/}'.